### DISCUSSION TITLE
捉个虫，有意想不到的排版问题

### DISCUSSION THREAD

**Discussion by Alossom** [amll-dev/applemusic-like-lyrics#213] [General]
![Uploading 1.jpg…]()
歌曲信息选项里开启显示专辑名称后，如果专辑名过长，音量调节bar会被挤到界面外

**Comment by Linho1219** [comment]
AMLL bncm 插件已停止维护，最终版本存档于 <https://github.com/amll-dev/applemusic-like-lyrics-proto>。

AMLL Player 中未能复现此问题，可能已经修复。

现关闭此讨论，敬请谅解。若仍遇此类问题请 reopen 或创建新条目。
